### PR TITLE
delete arrow in the 3 column

### PR DESCRIPTION
**Pull request by ndufreche**
Removing the 3 arrow from the template and add some translations.

Issue related #177 and #176

### CODE REVIEW

File: src/tb/apps/content/controllers/main.controller.js [hunk 1]
```diff
@@ -48,6 +48,8 @@ define(
     ) {
         'use strict';
 
+        var trans = Core.get('trans') || function (value) {return value;};
```
Comment (mickaelandrieu) [inline src/tb/apps/content/controllers/main.controller.js]: humm .. how does it work ? :)

Comment (ndufreche) [inline src/tb/apps/content/controllers/main.controller.js]: Yes it work

Comment (mickaelandrieu) [inline src/tb/apps/content/controllers/main.controller.js]: I'm sure it works, but "How" ? for information purposes

Comment (ndufreche) [inline src/tb/apps/content/controllers/main.controller.js]: Translation component expose his method trans on core api during his init

File: src/tb/apps/content/controllers/main.controller.js [hunk 2]
```diff
@@ -48,6 +48,8 @@ define(
     ) {
         'use strict';
 
+        var trans = Core.get('trans') || function (value) {return value; };
```
Comment (fkroockmann) [inline src/tb/apps/content/controllers/main.controller.js]: Why you test this function, if you use a Translator, you don't have problem.

**Pull request merged**